integration/integration.go

```diff
@@ -88,12 +88,12 @@ func (i *Integration) DefaultEntity() *Entity {
 	defer i.locker.Unlock()
```

Comment:
Do we want to keep `DefaultEntity()` method?

Reply:
nope! nice one